node_modules/col-users/lib/constants.js

```diff
@@ -24,6 +24,6 @@
  */
 
 var Constants = module.exports = {
-  'BASIC_USER_FIELDS': ['id', 'canvas_user_id', 'canvas_course_role', 'canvas_enrollment_state', 'canvas_full_name', 'canvas_image'],
+  'BASIC_USER_FIELDS': ['id', 'canvas_user_id', 'canvas_course_role', 'canvas_enrollment_state', 'canvas_full_name', 'canvas_image', 'canvas_email'],
```

Comment:
Does this mean that we're potentially exposing the email address in comments, etc. ? We should have a test that ensures we're not exposing the email address to other users.